assets/src/edit-story/components/library/panes/elements/elements.svg

```diff
@@ -0,0 +1,11 @@
OEAoLGxEUFgjbKY3A7yQ91HXdT2IbOymCYXDf9w9IGv5js9tjkzXUhhAKCRsRFBIE2yiRwGuSHu667vFTmzOJYJik7/tPSPoWnwyOkCKEIxz5TxBBfuas6ILA8Engk13Xfd+yW7MIhsn6vv+QpK9LutsyeUN9EMJGYSOCjcCzbMkEXpT0ma7rzH9RevK3huYq3E/8kCTzAnNzVXad3zKqLFDKgYBTAsOzefiZwFnP6LM+EQQw+3+FfVzS30p6a7jO8YoAnxCuUKz7gk8E6/JldjcEXpf0JUk/mPsV0VglF4kgTNj3/QckfUHScOTrJgGEcJPJolcQwaI4mcwngSckfbnruuF40VeSCIYV92/Ed0v6a0nvumgX9Q9CCCtljAhWAsu0HgjckvQVSc9e8ingsMBkEYTJ9m/I90j6HEIIVG4cEcINJGkXEEEaP0a7JPBLSY9JejpVAKH6xUQQJkQIgUT0iBCieOyNiMDOip7uCSwugEBkcRGEiRFCIBE9IoQontONiOA0I3q4J7CaAAKZ1UQQFkAIgUT0iBCieOYbEcE8G1rcE1hdAIHQ6iIICyGEQCJ6RAhRPDcbEcFNJlxxTyCbAAKpbCIICyKEQCJ6RAhRPNeNiOCaBa/cE8gugEAsuwjCwgghkIgeEUIUz9WvLz93ohvNECiZwGYCCFA2E0HYAEIIJKJHhDCDh08EM2C47IHA5gIIkDYXQdgIQggkokeEMMKDCEZAOPVAoBgBBFjFiCBsCCEEEtEjQtjjQQTR+4TGsggUJ4CApzgRhI0hhEAiemxeCIggen/QWAaBYgUQ8BQrgrBBhBBIRI/NCgERRO8LGrclULwAAp7iRRA2ihACieixOSEgguj9QOM2BNwIIOBxI4KwYYQQSESPzQgBEUTvAxrzEnAngIDHnQjCxhFCIBE9Vi8ERBDNn8Y8BNwKIOBxK4JQAEIIJKLHaoWACKK507guAfcCCHjciyAUghACieixOiEggmjeNK5DoBoBBDzViCAUhBACieixGiEggmjONC5LoDoBBDzViSAUhhACiejRvRAQQTRfGpchUK0AAp5qRRAKRAiBRPToVgiIIJorjWkEqhdAwFO9CEKhCCGQiB7dCQERRPOk8TICzQgg4GlGBKFghBBIRI9uhIAIojnSeB6B5gQQ8DQnglA4QggkosfihYAIovnRaCPQrAACnmZFEAAghEAieixWCIggmhuNcQLNCyDgaV4EAQRCCCSix+KEgAiiedE4TQABjLggghEQhDACMn1ajBAQwXRAXJ0kgAAmsUiIYAYMQpgBc3x5cyEgguNAOJskgAAmsVxfRATXLCZfIYRJLOOLmwkBEYyj4PyAAAI4gBF7iQhidA7aEMIBjPmX2YWACObDaLgFAZwZPiI4ExhCMAHLJgREYMqjlU4I4MKkEcGF4BCCCdzqQkAEphxq74QAEhNGBIkAEYIJ4GpCQAQm/rV2QgALJYsIFgKJEEwgFxcCIjBxr60TAlg4UUSwMFCEYAK6mBAQgYl3LZ0QwEpJIoKVwCIEE9hkISACE2fvnRDAygkigpUBIwQT4IuFgAhMfL12QgCZkkMEmUAjBBPos4WACExcvXVCAJkTQwSZgSMEE3CzEBCBiaeXTghgo6QQwUbgEYIJ/EkhIAITx9I7IYCNE0IEGweAEEwB/KOkr0t6teu6W4cjEMEhDXevEUAhkSGCQoJACKYgbnxCQAQmbqV1QgCFJYIICgsEIZgCuRLCvvdzplF02poAAtg6gZn1EcEMmK0vIwRTAkEIT5p602krAghgK/LGdRGBEdRW3RDCVuRZdwECCGABiDmmQAQ5KC+wBkJYACJT5CKAAHKRXmgdRLAQyFzTIIRcpFnnAgII4AJoJQxBBCWkcMEeEMIF0BiyFgEEsBbZTPMigkyg11oGIaxFlnkNBBCAAZKHLojAQ0qGPSIEAyS6LEUAASxFspB5EEEhQSy1DYSwFEnmmSCAACag1HAJEdSQ4kQNCGECCpcuJYAALiXnZBwicBLUpdtECJeSY5wkBNDIbYAIGgkaITQS9DJlIoBlOLqZBRG4iWqZjSKEZThWOgsCqDTYU2UhglOEKm1HCJUGe1lZCOAybtWMQgTVRHlZIQjhMm6VjEIAlQSZWgYiSCVYyXiEUEmQtjIQgI1TM70QQTNR2wpFCDZOTnshAKfBrb1tRLA2YafzIwSnwU1vGwFMc+HqngAi4FaIEkAIUTylNyKA0hMqZH+IoJAgSt8GQig9oaP9IYAjHJycIoAIThGi/YgAQjjCUdoJAigtESf7QQROgiptmwihqEQQQFFx+NsMIvCXWVE7RheTQ1AAAAHbSURBVAibxoEANsVfz+KIoJ4sN60EIWTFjwCy4q5/MURQf8ZZK0QIq+JGAKvibXdyRNBu9qtWjhAWxYsAFsXJZGMCiGBMhPNFCSCEJJwIIAkfg60EEIGVFP2SCCCEs/AhgLNw0TmVACJIJcj4swgghCguBBDFQ+NaBBDBWmSZN0oAIRzhQQBHODjJTQAR5CbOekcEGhcCAji6GzjZigAi2Io86x4RaEwICOAofU62JoAItk6A9Y8IVC4EBHCUNielEEAEpSTBPo4IVCYEBHCULielEUAEpSXCfo4IOBcCAjhKk5NSCSCCUpNhX0cEnAkBARylx0npBBBB6QmxvyMChQsBARylxYkXAojAS1Ls84hAYUJAAEfpcAIBCEAgI4FBCH3ff7bv++f7/F/DmsPa92QsmaUgsDgBPhEsjpQJtyCwfxjfJ+nTku5feQ9PSfqGpGe6rru18lpMD4HVCSCC1RGzQE4CeyHcKeljkj4q6e6F1n9R0o8k/VDSywhgIapMUwQBRFBEDGxiDQJ7Kdwl6X2Shk8L7zpDDMOD/wVJ/ynp55Je4uG/RkrMWQIBRFBCCuwhC4G9GG6TNHxieIekt0u6fb/4a5JelfSb4V/8kt7gwZ8lFhYpgMD/A13v1zecST7sAAAAAElFTkSuQmCC"/>
```

Comment:
I don't really know why this icon is an image and not vector, but I guess that's for future optimizations.